src/main/scala/mesosphere/marathon/api/LeaderProxyFilter.scala

```diff
@@ -216,11 +216,18 @@ class JavaUrlConnectionRequestForwarder(
         case "GET" | "HEAD" | "DELETE" =>
           leaderConnection.setDoOutput(false)
         case _ =>
+          // If Content-Type header is not set, and true is passed to setDoOutput,
+          // the header will be set to application/x-www-form-urlencoded.
```

Comment:
this is some kind of default of the underlying library we use?

Reply:
Yes.